### PR TITLE
Aquilon: add a SCDB migration guide

### PR DESCRIPTION
**Pull request by jouvin**
Attempt to summarize the main steps involved when migrating from SCDB

### CODE REVIEW

**Comment by jouvin** [comment]
I'm in favor of merging this PR once it has been reviewed. SCDB migration guide is still at early stage but is providing the most important hints, I think. We can always complete it later, with more feedback from SCDB sites...

**Comment by jouvin** [comment]
Current spellchecker failure is caused by the missing words in the dictionnary being defined in #251. I'll try to optimize both PRs but I would also like to avoid unnecessary conflicts.

File: _aquilon/scdb_migration.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+`site/global_variables.pan` template. Typically, this template can be imported in Aquilon after
+removing the `NETWORK_PARAMS` variable if present. Also the recommended directory is `site/config`
+rather than `site`.
+
+After the initial import of SCDB global variables, it is recommended to review the variable definitions
+that can be replaced by an Aquilon [service][aquilon_details]. This is in particular the case for
+the variable `AII_OSINSTALL_SRV ` who definition may be moved in the archetype `final.pan` template
+and be based on the `bootserver` service instance used for the host with the following Pan code:
+
+```bash
```
Comment (jrha) [inline _aquilon/scdb_migration.md]: I believe ` ```pan` should work here...

Comment (jouvin) [reply _aquilon/scdb_migration.md]: Good catch!

**Pull request merged**